app/views/home/index.html.erb

```diff
@@ -14,5 +14,33 @@
       <p>Looks like you don't have any poems yet.</p>
     <% end %>
     <%= link_to t("poems.actions.new"), new_poem_path, class: "btn button" %>
+    <h2>Your Submissions</h2>
+    <% if @submissions.present? %>
+      <table>
+        <thead>
+          <tr>
+            <th>Status</th>
+            <th>Submission Title</th>
+            <th>Submitted On</th>
+            <th>Submitted To</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @submissions.each do |submission| %>
+            <tr>
+              <td>submission.status</td>
+              <td>submission.title</td>
+              <td>submission.submission_date</td>
+              <td>submission.recipient</td>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
+    <% end %>
+    <%= 
+      link_to t("submissions.actions.new"),
```

Comment:
like this (1):
```
    <%= link_to t("submissions.actions.new"),
       new_submission_path,
       class: "btn button"
    %>
```
like (2)

```
    <%= link_to t("submissions.actions.new"),
            new_submission_path,
            class: "btn button"
    %>
```
or like (3)
```
    <%= link_to t("submissions.actions.new"),
              new_submission_path,
              class: "btn button"
    %>
```
Enabling Hound will help, but there are indentation norms that I still mix up. (1) looked weird to me with the path starting to the left of the `link_to` call, so I did this initially. I'd say 2 looks cleanest.